CONTRIBUTORS.md

```diff
@@ -42,7 +45,7 @@
 
 - [Harun Adil](https://github.com/HarunAdil)
 
-_ [Ramon R](https://github.com/artboycat)
+\_ [Ramon R](https://github.com/artboycat)
```

Comment:
I think you have mistake regarding writing your name in contributor list, you need use "-" this character while writing your name and use "\" character
![Screenshot 2024-05-31 210044](https://github.com/zero-to-mastery/start-here-guidelines/assets/169292446/1e1a66f2-f77e-4983-8e53-c44c34380535)

Reply:
Make this changes or you can follow instructions